docs/docs/Agents/agents.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Use Langflow Agents
+slug: /agents
+---
+
+import Icon from "@site/src/components/icon";
+
+Agents use LLMs as a brain to autonomously analyze problems and select tools to solve them.
+
+Langflow's [Agent component](/components-agents#agent-component) simplifies agent configuration so you can focus on application development.
+
+The Agent component provides everything you need to create an agent, including multiple LLMs, custom instructions, and tool configuration.
+
+## Agent settings
+
+You can configure the Agent component to use your preferred provider and model, custom instructions, and tools.
+
+### Agent models and providers
+
+Use the *Model Provider* and *Model Name* settings to select the LLM that you want the Agent to use.
+
+You must provide an authentication key for the selected provider, such as an OpenAI API key for OpenAI models.
+
+### Agent instructions and input
+
+In the *Agent Instructions* field, you can provide custom instructions that you want the Agent to use for every conversation.
+
+These instructions are applied in addition to the *Input*, which is provided at runtime.
+
+### Agent tools
+
+Agents are most useful when they have the appropriate tools available to complete requests.
+
+An Agent component can use any Langflow component as a tool, as long as you attach it to the Agent component.
+
+When you attach a component as a tool, you must configure the component as a tool by enabling *Tool Mode*.
+
+For more information, see [Configure tools for agents](/)
+
+## Use the Agent component in a flow
+
+:::tip
+For a pre-built demonstration, open the **Simple Agent** starter flow and follow along.
+:::
+
+Create an agent in Langflow, starting with the **Agent** component and working outward.
+
+1. Click **New Flow**, and then click **Blank Flow**.
+2. Click and drag an **Agent** component to your workspace.
+3. Use the default model or select another provider and model. If you want to use the default model, you need an OpenAI API key to run the flow.
+4. Add **Chat input** and **Chat output** components to your flow, and connect them to the tool calling agent.
+
+![Chat with agent component](/img/agent-example-add-chat.png)
+
+This basic flow allows you to chat with the agent in the **Playground**, but you're only chatting with the OpenAI LLM.
+To unlock the power of the Agent component, connect some tools.
+
+5. Add the **News Search**, **URL**, and **Calculator** components to your flow.
+6. Enable **Tool Mode** in the **News Search**, **URL**, and **Calculator** components.
+To enable **Tool Mode** in a component, in the component's configuration tab, click **Tool Mode**.
```

Comment:
59 and 60 are a bit redundant. In the MCP PR, I kind of avoided this awkward instruction by linking to the Component overview page where it introduces the component menu/tab.

Reply:
Good idea, added